From 4fb79ffad49058fa65fe27f081b06af8668a6577 Mon Sep 17 00:00:00 2001
From: Tom Hochstein <tom.hochstein@nxp.com>
Date: Mon, 10 Aug 2026 12:35:14 -0700
Subject: [PATCH] Fix prime_tests typo

Fix the build break:
```
| /.../src/prime_tests/load_hmac_key.c:21:28: error: 'CRYPTO_AUTH_SHA_521' undeclared (first use in this function); did you mean 'CRYPTO_AUTH_SHA_512'?
|    21 |         uint8_t sha_algo = CRYPTO_AUTH_SHA_521;
|       |                            ^~~~~~~~~~~~~~~~~~~
|       |                            CRYPTO_AUTH_SHA_512
```

Upstream-Status: Pending
Signed-off-by: Tom Hochstein <tom.hochstein@nxp.com>
---
 src/prime_tests/hmac.c          | 2 +-
 src/prime_tests/load_hmac_key.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/prime_tests/hmac.c b/src/prime_tests/hmac.c
index caed6ba..abc9ee0 100644
--- a/src/prime_tests/hmac.c
+++ b/src/prime_tests/hmac.c
@@ -16,7 +16,7 @@
 static uint8_t sha_algo[NUM_KEY_SIZE] = {
 	CRYPTO_AUTH_SHA_256,
 	CRYPTO_AUTH_SHA_384,
-	CRYPTO_AUTH_SHA_521,
+	CRYPTO_AUTH_SHA_512,
 };
 
 static uint16_t key_sz[NUM_KEY_SIZE] = {
diff --git a/src/prime_tests/load_hmac_key.c b/src/prime_tests/load_hmac_key.c
index 67f7ead..bf615c7 100644
--- a/src/prime_tests/load_hmac_key.c
+++ b/src/prime_tests/load_hmac_key.c
@@ -18,7 +18,7 @@ int load_hmac_key(void)
 	uint32_t i = 0;
 	uint8_t hmac_key[KEY_SZ] = {0};
 	open_service_args_t op_serv_args = {0};
-	uint8_t sha_algo = CRYPTO_AUTH_SHA_521;
+	uint8_t sha_algo = CRYPTO_AUTH_SHA_512;
 
 	/* open prime service */
 	op_serv_args.serv_type = SERVICE_TYPE_ALL;
-- 
2.43.0

